sync_radar_data_globus.py

```diff
@@ -121,7 +122,7 @@ def __init__(self, client_id, client_secret=None, transfer_rt=None):
         self.sync_pattern = args.sync_pattern
         self.data_type = args.data_type
         self.sync_local_dir = args.sync_local_dir
-        self.mirror_root_dir = "~/chroot/sddata"
+        self.mirror_root_dir = "/chroot/sddata"
```

Comment:
Sorry, can you please explain why you took out `~` (in this comment, I am more just curious if something changed with cedar vs. globus)?

Reply:
It changed when it went from graham to cedar, not sure why...